urgent-care-tests/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>urgentcare-apis-parent</artifactId>
+    <groupId>gov.va.api.health</groupId>
+    <version>1.0.40-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>urgent-care-tests</artifactId>
+  <version>1.0.40-SNAPSHOT</version>
+  <properties>
+    <docker-maven-plugin.version>0.24.0</docker-maven-plugin.version>
+    <docker.organization>vasdvp</docker.organization>
+    <docker.tag>latest</docker.tag>
+  </properties>
+  <profiles>
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <version>${docker-maven-plugin.version}</version>
+            <executions>
+              <execution>
+                <id>build-test</id>
+                <configuration>
+                  <images>
+                    <image>
+                      <name>${docker.organization}/health-apis-${project.artifactId}</name>
```

Comment:
This is a reminder that we'll need an accompanying PR to update the name of the test artifact we use in the DU. Right now I think we just use "urgent-care-tests"